core/src/main/java/com/sequenceiq/cloudbreak/init/blueprint/BlueprintLoaderService.java

```diff
@@ -42,17 +43,23 @@
     @Inject
     private ClusterTemplateService clusterTemplateService;
 
+    private Map<String, Blueprint> defaultBlueprints;
+
+    @PostConstruct
+    public void getBlueprintsFromCache() {
+        defaultBlueprints = Map.copyOf(defaultBlueprintCache.defaultBlueprints());
```

Comment:
defaultBlueprints are also initialized there in postconstruct. Can you please explain why it helps ?

Reply:
getting defaultBlueprints is kind of slow due to :
        defaultBlueprints.forEach((key, value) -> result.put(key, SerializationUtils.clone(value)));

but this list is not meant to change, so it is okay to get it only once (at startup), right?

Reply:
yep